client/src/API/AuthMethods.ts

```diff
@@ -1,31 +1,67 @@
-import axios from "axios";
+import axios, { AxiosError } from "axios";
 import { LoginData, UserInfoData, URL } from "../constants";
+import { useHistory } from "react-router-dom";
+
+const history = useHistory();
 
 export const signUp = async (userInfo: UserInfoData) => {
   try {
-    await axios.post(`${URL}/auth/signUp`, {
-      email: userInfo.email,
-      password: userInfo.password,
-      first_name: userInfo.firstName,
-      last_name: userInfo.lastName,
-    });
-  } catch (e: any) {
-    // Will need to research more on how to better type the error
-    if (e.response.status > 399) {
-      alert(e.response.data.message[0]);
+    const response = await axios.post(
+      `${URL}/auth/signUp`,
+      {
+        email: userInfo.email,
+        password: userInfo.password,
+        first_name: userInfo.firstName,
+        last_name: userInfo.lastName,
+      },
+      {
+        withCredentials: true,
+        headers: {
+          Authorization: `Bearer ${sessionStorage.getItem("authToken")}`,
+        },
+      }
+    );
+    sessionStorage.setItem("authToken", response.data.accessToken);
+    if (response) {
+      history.push("/");
     }
+  } catch (e) {
+    const err = e as AxiosError;
+    console.log(err.response?.data?.message);
```

Comment:
Here, we may want to put an alert. If they receive an error, it'll most likely be that the email has already been used or something is wrong with the server. Maybe console.log and experiment with what comes back so we can decide how to show errors to the user.